src/main/java/edu/harvard/iq/dataverse/FilePage.java

```diff
@@ -832,7 +832,7 @@ public boolean isPreviewAllowed() {
             // Always allow preview for PrivateUrlUser
```

Comment:
I updated the code as suggested which of course makes sense @scolapasta. 

Perhaps checking to ensure preview of embargoed unrestricted files (only those with permissions to access it can see it?) works as the Dataverse team expects would be a good idea before merging.

Let me know if further changes are required...

Reply:
I have to test it further before merging so if you leave it in community dev, I will let you know when it's good to go.

Reply:
It's now ready to go. The previewTab wasn't rendering in my code because I had an extra } in my code.
I can confirm that I have tested it and that the previewTab now shows for restricted files that a user has permission to download.
The dataverse team should probably test it with an unrestricted but actively embargoed file to make sure the behaviour is what the Dataverse team expects.